upgrade/Stage_0_Prerequisites.md

```diff
@@ -140,14 +140,14 @@ Stage 0 has several critical procedures which prepare the environment and verify
 
    ```text
    ##################################################################################
-   # CSM version:  1.2
-   # CANU version: 1.6.5
+   # CSM version:  1.3
+   # CANU version: 1.6.14
    ##################################################################################
    ```
 
    - Output like the above text means that the switches have a CANU-generated configuration for CSM 1.2 in place. In this case, follow the steps in
      [Management Network 1.2 to 1.3](../operations/network/management_network/1.2_to_1.3_upgrade.md).
-   - If the banner does NOT contain text like the above, then contact support in order to get `CSM 1.2 switch configuration` applied to the system.
+   - If the banner does NOT contain text like the above, then contact support in order to get `CSM 1.3 switch configuration` applied to the system.
```

Comment:
change to: "If the banner does NOT contain the appropriate CSM version, then apply the `CSM 1.3 switch configuration` or contact support.